packages/worker/src/constants/templates/tests/templates.spec.ts

```diff
@@ -0,0 +1,177 @@
+import { TestConfiguration } from "../../../tests"
+import { addBaseTemplates, loadTemplateConfig } from ".."
+import { EmailTemplatePurpose, type Template } from "@budibase/types"
+import { join } from "path"
+
+describe("Loading yaml email templates", () => {
+  const config = new TestConfiguration()
+
+  let templates: Template[] = []
+
+  beforeAll(async () => {
+    addBaseTemplates(templates, "email")
+    // Core is not an updatable template and should never be included
+    templates = templates.filter(t => t.purpose !== EmailTemplatePurpose.CORE)
+
+    await config.beforeAll()
+  })
+
+  afterAll(async () => {
+    await config.afterAll()
+  })
+
+  it("should ignore an invalid path", async () => {
+    const consoleSpy = jest.spyOn(console, "log").mockImplementation(() => {})
```

Comment:
Could extract this out into a beforeEach and the `consoleSpy.mockRestore()` into an afterEach as they're used in all tests as far as I can tell. Is minor, though, ignore if you don't feel like it.